src/App.js

```diff
@@ -1,29 +1,42 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Button from './Calculator/Button';
+import Screen from './Calculator/Screen';
+import ButtonCard from './UI/ButtonCard';
+/* eslint-disable react/destructuring-assignment */
```

Comment:
as per the requirements you are required to import the react components as so `import React, { Component } from 'react';`